django/core/checks/security/base.py

```diff
@@ -116,6 +116,20 @@
     id='security.W022',
 )
 
+W023 = Warning(
+    "One of your SECRET_KEY_FALLBACK values has less than %(min_length)s "
+    "characters, less than %(min_unique_chars)s unique characters, or it's "
+    "prefixed with '%(insecure_prefix)s' indicating that it was generated "
+    "automatically by Django. Please generate a long and random value, "
+    "otherwise many of Django's security-critical features will be vulnerable"
+    " to attack." % {
+        'min_length': SECRET_KEY_MIN_LENGTH,
+        'min_unique_chars': SECRET_KEY_MIN_UNIQUE_CHARACTERS,
+        'insecure_prefix': SECRET_KEY_INSECURE_PREFIX,
+    },
+    id='security.W023',
+)
```

Comment:
So, I've been hemming and hawing on whether to mention it, because it conceptually works when in the error message, but it still seems slightly _'off'_ to me that the warning would say `SECRET_KEY_FALLBACK` when the setting is `SECRET_KEY_FALLBACKS` (plural). I guess if we're not going to say _which_ one errored (which _we could_, using hints) I think it'd make more sense to say `One of your SECRET_KEY_FALLBACKS has less ...`

Reply:
Good catch :dart: I didn't notice this :eagle: Definitely `SECRET_KEY_FALLBACKS`.

Reply:
Addressed in 7932631d

Reply:
I wrote up a version of a way to include a hint indicating which key caused the warning. I couldn't figure out how to cause an `AttributeError` or `ImproperlyConfigured` exception to test the `invalid_key=None` case though. I can revert this if it's too much.

One concern would be leaking the key. Maybe I should have it indicate the index? If so, do we use base 0 or 1 counting?

Reply:
I added test for `AttributeError` and pushed edits to system checks.